data/dbModels/organizations.js

```diff
@@ -27,13 +27,23 @@ async function create(organization) {
   }
 }
 
-function read(slack_org_id = null) {
-  if (slack_org_id) {
-    return db('organizations')
-      .where({ slack_org_id })
-      .first();
+async function read(slack_org_id = null, org_id = null) {
+  try {
+    if (slack_org_id) {
+      const org = db('organizations')
```

Comment:
Await an asynchronous operation to resolve before returning the result, to avoid bug of returning `null` or `undefined`